pyproject.toml

```diff
@@ -14,9 +14,7 @@ exclude = '''
 )
 '''
 [tool.pytest.ini_options]
-flake8-max-line-length = 210
-flake8-ignore = ["E203", "W503", "D100", "D101", "D102", "D103", "D104", "D105", "D107", "legacy.py ALL"]  # See https://github.com/PyCQA/pycodestyle/issues/373
-addopts = "--flake8 --mypy --isort"
+addopts = "--mypy"
```

Comment:
Not sure how pytest-black-ng works (will test it), but for ruff I tested it, you don't need any arguments, if you have pytest-ruff installed, it will check it

Reply:
I added --black and --ruff to be safe - docs says you should

Reply:
pipeline output looks good, e.g.:
 flair/datasets/base.py::ruff PASSED                                      [  2%]
flair/datasets/base.py::mypy PASSED                                      [  2%]
flair/datasets/base.py::black PASSED                                     [  2%]